source/compilation/builtins/SystemTasks.cpp

```diff
@@ -608,6 +608,58 @@ class SdfAnnotateTask : public SystemTaskBase {
     }
 };
 
+class PlaTask : public SystemTaskBase {
+public:
+    PlaTask(const std::string& name) :
+        SystemTaskBase(name), requiredArgs(3) {};
```

Comment:
No need for the member variable if this is always going to be constant. Can just put it right in the checkArgCount call or make it a local constant there.